packages/amplify-opensearch-simulator/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@aws-amplify/amplify-opensearch-simulator",
+  "version": "1.0.0",
+  "description": "Opensearch local simulator",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/aws-amplify/amplify-cli.git",
+    "directory": "packages/amplify-opensearch-simulator"
+  },
+  "author": "Amazon Web Services",
+  "license": "Apache-2.0",
+  "main": "lib/index.js",
+  "keywords": [
+    "graphql",
+    "appsync",
+    "aws"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "clean": "rimraf lib tsconfig.tsbuildinfo node_modules",
+    "test": "jest"
+  },
+  "dependencies": {
+    "amplify-cli-core": "3.3.0",
+    "amplify-prompts": "2.6.0",
+    "aws-sdk": "^2.1233.0",
+    "detect-port": "^1.3.0",
+    "execa": "^5.1.1",
+    "fs-extra": "^8.1.0",
+    "get-port": "^5.1.1",
+    "gunzip-maybe": "^1.4.2",
+    "node-fetch": "^2.6.7",
+    "openpgp": "^4.10.10",
+    "promise-toolbox": "^0.20.0",
+    "tar": "^6.1.11",
+    "wait-port": "^0.2.7",
+    "which": "^2.0.2"
+  },
+  "devDependencies": {
+    "@types/node": "^12.12.6",
+    "@types/openpgp": "^4.4.18",
+    "hash.js": "^1.1.7",
```

Comment:
If using `opengpg`, `hash.js` is probably not needed. I cannot find any usage of it in this PR.